src/python/grpcio/grpc/__init__.py

```diff
@@ -833,7 +833,11 @@ class HandlerCallDetails(six.with_metaclass(abc.ABCMeta)):
 
 
 class GenericRpcHandler(six.with_metaclass(abc.ABCMeta)):
```

Comment:
True, but it's within reason to make the new service specification accessible with a "rerun your code generation" requirement? I agree that it's not as nice as a completely seamless integration but remember that the primary use case in the long view is for code generation to be integrated into [Bazel](http://www.bazel.io/) and done more or less continuously during development.